### PR TITLE
last commit 

### PR DESCRIPTION
**Pull request by benedictatac**
has minor changes, added a database too

### CODE REVIEW

File: client/client.py [hunk 1]
```diff
@@ -1,44 +1,58 @@
...
-
-# Might need to instantiate TCP socket here as well
-
+TCP_PORT = 5006  # Separate port for TCP
+UDP_PORT = 5005
+serverAddr = (SERVER_IP, UDP_PORT)
 
+# Create UDP and TCP sockets
 clientSocket = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
-isRegistered = False
```
Comment (peach3es) [inline client/client.py:15]: keep the isRegistered variable

File: client/client.py [hunk 2]
```diff
@@ -95,4 +110,5 @@ def searchMenu(name):
     registrationMenu()
 
     # Close the socket when done
-    clientSocket.close()
```
Comment (peach3es) [inline client/client.py:98]: keep clientSocket.close() & remove udpSocket.close(). the variable udpSocket is just a user prompted string value not a socket value. I'd suggest differentiating the variable names for tcpSocket. let's keep the tcpSocket on line 9 and change the two user prompted sockets to udpSocketValue and tcpSocketValue (check image below).
image

File: server.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import socket
```
Comment (peach3es) [inline server.py:1]: why is there another server,py in the root folder? is this like your test file?

**Review by peach3es — CHANGES_REQUESTED** [review]
check the comments first and re request when you're done

File: server.py (hunk 3 above)
Comment (benedictatac) [reply server.py:1]: yeah that was a testing file i forgot to delete. Deleting it now

**Review by peach3es — APPROVED** [review]
should be good to merge now, make sure to resolve any merge conflicts if there are any, lmk if you need help with that.

**Pull request merged**